docs/reference/snapshot-restore/apis/get-snapshot-api.asciidoc

```diff
@@ -76,6 +83,78 @@ If `true`, returns all available information about a snapshot. Defaults to `true
 +
 If `false`, omits additional information about the snapshot, such as version information, start and end times, and the number of snapshotted shards.
 
+[role="child-attributes"]
```

Comment:
@original-brownbear, can you please review the response body parameters that I added?